Source/DafnyCore/Options/EnforceDeterminismOption.cs

```diff
@@ -0,0 +1,17 @@
+namespace Microsoft.Dafny;
+
+public class EnforceDeterminismOption : BooleanOption {
```

Comment:
I think you will need to run `make options` in docs/DafnyRef and then commit/push the new Options.txt file, or else the CI building of DafnyRef.pdf will fail.

Reply:
It did not fail, which must mean that the tests are not checking these new options -- so that should be fixed.

Reply:
And to be honest, I really don't like this split between old and new help. If I were a newish user it would be very confusing.

Reply:
The `Options.txt` file contains the old help, which is not changed by this PR, so there's no need to update it.